WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -505,4 +506,12 @@ public void onEventMainThread(NotificationEvents.NoteModerationFailed event) {
 
         EventBus.getDefault().removeStickyEvent(NotificationEvents.NoteModerationFailed.class);
     }
+
+    public SiteModel getSelectedSite() {
+        if (getActivity() instanceof WPMainActivity) {
+            WPMainActivity mainActivity = (WPMainActivity) getActivity();
+            mainActivity.getSelectedSite();
```

Comment:
> I'll pass the site to that fragment instead of having this method.

Ooops, no actually this is a fragment living in the Tab, we need that getSelectedSite() method, and we need to fix it.